src/template.ts

```diff
@@ -1,12 +1,21 @@
-import Ajv = require('ajv');
-import stringify = require('json-stable-stringify');
+import Ajv from 'ajv';
+import stringify from 'json-stable-stringify';
 import * as TJS from 'typescript-json-schema';
+import loadTsConfig from './loadTsConfig';
 
 export const TSLINT_DISABLE = `/* tslint:disable */`;
 export const GENERATED_COMMENT = `// generated by typescript-json-validator`;
 
 export const IMPORT_INSPECT = `import {inspect} from 'util';`;
-export const IMPORT_AJV = `import Ajv = require('ajv');`;
+
+export const IMPORT_AJV = (): string => {
+  const tsConfig = loadTsConfig();
```

Comment:
Can we pass `tsconfig` in as a parameter. It already gets loaded once in order to generate the json schema. We don't really want to load it again. That way you also won't need to default to `process.cwd()` and it would be easier to add a `--project` flag to explicitly select a tsconfig file.